spinalcordtoolbox/scripts/sct_run_batch.py

```diff
@@ -524,12 +563,17 @@ def reset_streams():
     if (arguments.include is not None) and (arguments.include_list is not None):
         parser.error('Only one of `include` and `include-list` can be used')
 
-    if (arguments.exclude is not None) and (arguments.exclude_list is not None):
-        parser.error('Only one of `exclude` and `exclude-list` can be used')
+    if sum([arg is not None for arg in [arguments.exclude, arguments.exclude_list, arguments.exclude_file]]) > 1:
+        parser.error('Only one of `exclude`, `exclude-list`, and `exclude-file` can be used')
+
+    # Check `-exclude-list` (or if `-exclude-file` contains subjects)
+    exclude_list = arguments.exclude_list
+    if arguments.exclude_file['subjects']:
+        exclude_list = arguments.exclude_file['subjects']
```

Comment:
Whoops, this line is causing an issue:

```
TypeError: 'NoneType' object is not subscriptable
```

Should be an easy fix:

```suggestion
    if arguments.exclude_file is not None and arguments.exclude_file['subjects']:
        exclude_list = arguments.exclude_file['subjects']
```